cartographer_ros/cartographer_ros/color.cc

```diff
@@ -14,20 +14,21 @@
  * limitations under the License.
  */
 
-#include "cartographer_ros/color_manager.h"
+#include "cartographer_ros/color.h"
 
 #include <cmath>
 
 #include "glog/logging.h"
 
-namespace cartographer_ros {
+namespace {
```

Comment:
Are you sure? E.g. in node.cc you have `kInfiniteSubscriberQueueSize` inside an anonymous namespace, but are using it later inside functions that are inside `::cartographer_ros`.